<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CADASTRO DE CLIENTES</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>641</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_add">
       <property name="text">
        <string>Adicionar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_editar">
       <property name="text">
        <string>Editar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_excluir">
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="list_dados">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>41</y>
      <width>641</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CLIENTES CADASTRADOS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>754</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
